entc/integration/gremlin/ent/fieldtype.go

```diff
@@ -17,8 +17,9 @@ import (
 	"entgo.io/ent/dialect/gremlin"
 	"entgo.io/ent/entc/integration/ent/role"
 	"entgo.io/ent/entc/integration/ent/schema"
-	"entgo.io/ent/entc/integration/gremlin/ent/fieldtype"
 	"github.com/google/uuid"
+
+	"entgo.io/ent/entc/integration/gremlin/ent/fieldtype"
```

Comment:
In such cases, the generated package imports a "local package". Therefore, the grouping should be based on the Go modules (project root). In this case, it's `entgo.io/ent`. i.e. if we have 2 import paths:
```go
<root-pkg>/ent/user
<root-pkg>/pkg/types
```

We want to set `-local` to `<root-pkg>`, and not `<root-pkg>/ent`, right?

Reply:
Ahm, yeah. `goimports` needs an option to add a 4th section :-D

Reply:
Yeah, I agree😁. How to get the `<root-pkg>`?

Reply:
Another idea, `entc` will set `LocalPrefix` to `<root-pkg>` only if the `LocalPrefix` is empty.

Reply:
We can add some code for loading the Go module information (see my PR - https://github.com/ent/ent/pull/2175) and use it as a LocalPrefix, but I agree it should be set only if `LocalPrefix` is empty. Let me know what do you think.

Reply:
`Module.Path` should return the go module path. But again, tbh, I think there are no real guidelines on how package paths should be grouped. I, for example, like to group all my company/product import paths in the same block. Thoughts?

Reply:
I agree. I have already fixed my problem with `imports.LocalPrefix = "<config-goes-here>"`. This PR could be closed😂.

Reply:
Thanks for your solution @a8m !